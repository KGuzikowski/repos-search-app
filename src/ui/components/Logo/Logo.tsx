import React from 'react'
import styled from 'styled-components'

const LogoDiv = styled.div`
    font-size: 25px;
    color: #fff;
`

const Logo = () => (
    <LogoDiv>
        ReposSearch
    </LogoDiv>
)

export default Logo