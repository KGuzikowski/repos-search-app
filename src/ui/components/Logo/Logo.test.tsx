import React from 'react'
import { create } from 'react-test-renderer'
import Logo from './Logo'

it('renders Logo component correctly', () => {
    const component = create(<Logo />)
    expect(component.toJSON()).toMatchSnapshot()
})