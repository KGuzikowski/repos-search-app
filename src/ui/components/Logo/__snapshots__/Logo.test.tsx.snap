// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo component correctly 1`] = `
<div
  className="sc-AxjAm elAYzD"
>
  ForCallstack
</div>
`;
